src/components/_navbar.scss

```diff
@@ -106,17 +113,13 @@
       }
 
       .navbar-nav {
-        overflow: hidden;
-
-        &.collapsing {
-          @extend .collapse;
-        }
+        width: $navbar-width-sm;
 
         &.collapse {
           position: relative;
           left: px(50);
           top: px(-50);
-          width: $navbar-width - px(24);
+          width: $navbar-width - px(14);
```

Comment:
I did not find what is this `14px`. Can you help me out?

Reply:
When the navbar is expanded, if you have sub-items under a nav-item, they are indented a certain amount. When the navbar is collapsed, we show the sub-items as a fly-out menu on the right when the parent item is hovered. To make the sub-items appear exactly the same as they do in expanded mode, we need to decrease the width of the items to account for the indentation they have in expanded mode. This is the amount required to account for the indentation.